Framework/PythonInterface/plugins/algorithms/SCDAdjustSampleNorm.py

```diff
@@ -0,0 +1,246 @@
+        self.declareProperty(MultipleFileProperty(name="Filename",
+                                                  extensions=["_event.nxs", ".nxs.h5", ".nxs"],
+                                                  action=FileAction.OptionalLoad),
+                             doc="Input autoreduced detector scan data files to convert to Q-space.")
+        self.declareProperty(
+            FileProperty(name="VanadiumFile", defaultValue="", extensions=[".nxs"], direction=Direction.Input,
+                         action=FileAction.OptionalLoad),
+            doc="File with Vanadium normalization scan data")
+
+        # Alternative WS inputs
+        self.declareProperty("InputWorkspaces", defaultValue="", direction=Direction.Input,
+                             doc="Workspace or comma-separated workspace list containing input MDHisto scan data.")
+        self.declareProperty(IMDHistoWorkspaceProperty("VanadiumWorkspace", defaultValue="", direction=Direction.Input,
+                                                       optional=PropertyMode.Optional),
+                             doc="MDHisto workspace containing vanadium normalization data")
+
+        # Detector adjustment options
+        self.declareProperty("DetectorHeightOffset", defaultValue=0.0, direction=Direction.Input,
+                             doc="Optional distance to move detector height (relative to current position)")
+        self.declareProperty("DetectorDistanceOffset", defaultValue=0.0, direction=Direction.Input,
+                             doc="Optional distance to move detector distance (relative to current position)")
+
+        # Which conversion algorithm to use
+        self.declareProperty("OutputAsMDEventWorkspace", defaultValue=True, direction=Direction.Input,
+                             doc="Whether to use ConvertHFIRSCDtoQ for an MDEvent, or ConvertWANDSCDtoQ for an MDHisto")
+
+        # MDEvent WS Specific options for ConvertHFIRSCDtoQ
+        self.declareProperty(FloatArrayProperty("MinValues", [-10, -10, -10], FloatArrayLengthValidator(3),
+                                                direction=Direction.Input),
+                             doc="3 comma separated values, one for each q_sample dimension")
+        self.declareProperty(FloatArrayProperty("MaxValues", [10, 10, 10], FloatArrayLengthValidator(3),
+                                                direction=Direction.Input),
+                             doc="3 comma separated values, one for each q_sample dimension; must be larger than"
+                                 "those specified in MinValues")
+        self.declareProperty("MergeInputs", defaultValue=False, direction=Direction.Input,
+                             doc="If all inputs should be merged into one MDEvent output workspace")
+
+        # MDHisto WS Specific options for ConvertWANDSCDtoQ
+        self.declareProperty(FloatArrayProperty("BinningDim0", [-8.02, 8.02, 401], FloatArrayLengthValidator(3),
```

Comment:
These are good default values for WAND but I don't think they are so great for DEMAND, I will try to get some better defaults to use.